wormhole-connect/src/views/Bridge/Inputs/Inputs.tsx

```diff
@@ -202,7 +210,12 @@ function Inputs(props: Props) {
                 {/* balance */}
                 <div className={classes.balance}>
                   <Input label="Balance">
-                    <div>{props.balance || NO_INPUT}</div>
+                    <div
+                      style={{ cursor: 'pointer' }}
```

Comment:
It might make sense to create a class and add some other css to highlight the option and let the user know they can click on it (borders/shadows, background, etc)